src/test/java/giis/eshopcontainers/e2e/functional/common/BaseLoggedTest.java

```diff
@@ -0,0 +1,181 @@
+package giis.eshopcontainers.e2e.functional.common;
+
+import giis.eshopcontainers.e2e.functional.common.exceptions.ElementNotFoundException;
+import giis.eshopcontainers.e2e.functional.model.EShopUser;
+import giis.eshopcontainers.e2e.functional.utils.Click;
+import giis.eshopcontainers.e2e.functional.utils.Navigation;
+import io.github.bonigarcia.wdm.managers.ChromeDriverManager;
+import io.github.bonigarcia.wdm.managers.FirefoxDriverManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.logging.LogEntries;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.Properties;
+
+import static org.openqa.selenium.logging.LogType.BROWSER;
+
+public class BaseLoggedTest {
```

Comment:
The purpose and/or rationale of this class (and other main classes) should be documented